test/kube2e/upgrade/upgrade_utils.go

```diff
@@ -15,72 +14,46 @@ import (
 	"github.com/solo-io/go-utils/changelogutils"
 	"github.com/solo-io/go-utils/githubutils"
 	"github.com/solo-io/go-utils/versionutils"
+	"github.com/solo-io/skv2/codegen/util"
 )
 
-// GetUpgradeVersions for the given repo.
-// This will return the lastminor, currentminor, and an error
-// This may return lastminor + currentminor, or just lastminor and an error or a just an error
-func GetUpgradeVersions(ctx context.Context, repoName string) (lastMinorLatestPatchVersion *versionutils.Version, currentMinorLatestPatchVersion *versionutils.Version, err error) {
-
-	twoBack, currentBranch, curMinorErr := getLastReleaseOfCurrentMinor()
-	currentMinorLatestPatchVersion = twoBack
-	if errors.Is(curMinorErr, version.FirstReleaseError) {
-		// we are on the first release of a minor
-		// we should use this branch rather than the last release
-		currentMinorLatestPatchVersion = currentBranch
-	} else if curMinorErr != nil {
-		return nil, nil, curMinorErr
+// GetUpgradeVersions returns two semantic versions of a repository:
+//   - prevLtsRelease: the latest patch release of v1.m-1.x
+//   - latestRelease:  the latest patch release of v1.m.x
+//
+// Originally intended for use in upgrade testing, it can return any of:
+//   - (prevLtsRelease, latestRelease, nil): all release versions computable
+//   - (prevLtsRelease, nil, nil):           only prevLtsRelease computable
```

Comment:
```suggestion
//   - (prevLtsRelease, nil, nil):           only prevLtsRelease computable (ie current branch has never been released)
```
The inline comment explains this case, but I think it's helpful here as well
Feel free to reword